spotify-artist-viewer/Models/SAArtist.h

```diff
@@ -9,4 +9,16 @@
 
 @interface SAArtist : NSObject
 
+//artist name, photo, bio, spotify ID number
+
+@property (nonatomic)NSString *artistName;
+@property (nonatomic)NSString *artistUri;
+@property(nonatomic)NSString *imgURL;
+
+
+
```

Comment:
Try to minimize unnecessary whitespace.

See style guide: https://github.com/IntrepidPursuits/objective-c-style-guide#cgrect-functions